src/app/trade-creator-page/trade-creator-table/trade-creator-table.component.ts

```diff
@@ -13,6 +13,7 @@ export class TradeCreatorTableComponent {
 
   @Input() displayForCreatorForm: boolean = false;
   @Input() creatorCoinTrade: CreatorCoinTrade;
+  @Input() userFollowingCreator: boolean;
```

Comment:
instead of making this an input, should we just add the follow service in the constructor and compute this on init?